src/Domain/Heroes/StrengthUpdatedEvent.cs

```diff
@@ -0,0 +1,5 @@
+﻿using SSW.CleanArchitecture.Domain.Common.Base;
+
+namespace SSW.CleanArchitecture.Domain.Heroes;
+
+public record StrengthUpdatedEvent(Hero Hero) : DomainEvent;
```

Comment:
I don't think this should be an object reference, what if multiple things mutate the Hero object?

Most of the time events have the fields separately & are immutable.

Reply:
We could capture all properties at the time the event was raised to avoid possible unexpected aggregate updates.

Reply:
Done